packages/terra-compact-interactive-list/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added `hasVisibleBorders` prop.
```

Comment:
Can we add a the purpose of the prop as well? e.g. Added `hasVisibleBorders` prop to .....

Reply:
Updated in https://github.com/cerner/terra-framework/pull/2075/commits/27d253d7d771c7c99c81e0a05162723569d6ed08